doc/api/events.md

```diff
@@ -666,6 +666,9 @@ event or that is rejected when the `EventEmitter` emits `'error'`.
 The `Promise` will resolve with an array of all the arguments emitted to the
 given event.
 
+Note: This method is intentionally generic and works with web platform
```

Comment:
@mcollina WDYT? I'm not sure what's the correct way to do this in our docs is

Reply:
I would add a note about the difference in error handling.

Reply:
Done.

Reply:
Feel free to edit/push this